src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Row, ScoreNameCell, LevelCell } from './SubscoresBlock.style';
+import { orderedCcviThemes, CcviThemeInfo } from 'cms-content/tooltips';
+import { InfoTooltip } from 'components/InfoTooltip';
+import { RegionCcviItem } from 'common/data';
+import { getCcviLevel, getCcviLevelColor, getCcviLevelName } from 'common/ccvi';
+
+export interface ThemeWithScore extends CcviThemeInfo {
+  score: number;
+}
+
+const SubscoresBlock: React.FC<{ scores: RegionCcviItem }> = ({ scores }) => {
```

Comment:
I've been using 'theme' and 'subscore' interchangeably, which might be confusing?